packages/faustwp-core/src/components/WordPressTemplate.tsx

```diff
@@ -116,7 +119,10 @@ export function WordPressTemplate(props: WordPressTemplateProps) {
       let queryArgs: QueryOptions = {
         query: SEED_QUERY,
         variables: {
-          uri: seedQueryUri,
+          // Conditionally add relevant query args.
+          ...(!isPreview && { uri: seedQueryUri }),
+          ...(isPreview && { id: databaseId }),
+          ...(isPreview && { asPreview: true }),
```

Comment:
Interesting use of conditional spreading properties. I'll have to use this in the future.